Makefile.am

```diff
@@ -169,8 +169,8 @@ check-regression: tests/config-flags.pm
 		     rm -f tests/$$f; ln -s ../srctests/$$f tests; \
 		   done; fi ;; \
 	    esac; \
-	    echo "cd tests && $(PERL) $(PERLFLAGS) ./run_make_tests.pl -srcdir $(abs_top_srcdir) -make ../make$(EXEEXT) $(MAKETESTFLAGS)"; \
```

Comment:
@boretom note this with respect to your comment about `catfile`.

Also note that I couldn't figure out how to easily set $abs_top_srcdir to be in `src` rather than the top-level directory in the automess language. So I just added the `/src` directory suffix.